src/backend/revoker/scanAttestations.ts

```diff
@@ -70,7 +80,9 @@ export async function* scanAttestations() {
 
     // extract the parameters
     const params = parseParams(event).parsedParams;
-    const owner = Did.fromChain(params.AttesterOf as AccountId32);
+
+    const owner = getDidUriFromAccountHex(params.AttesterOf as HexString);
```

Comment:
Before this code was changed to use names, this line was as simple as that:

```ts
    const owner = Did.fromChain(params[0].value);

Reply:
True, this is simpler.  

`  const owner = Did.fromChain(params.AttesterOf as AccountId32);
`

Reply:
Changed to your suggestion.

Reply:
Wait, that was exactly what it was before this branch. It caused a bug on ctype-hub.